### ISSUE TITLE
NTR: Hexokinase activity low in RBC (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8227]
New term request Hexokinase activity low in RBC (elthat_130422040327): Erythrocytes, red blood cells

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000559
- label: Reduced erythrocyte hexokinase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:7234862
- def: Activity or concentration of hexokinase (EC 2.7.1.1) in red blood cells below the lower limit of normal. Arginase catalyzes the last step of the urea cycle.
- comment: Hexokinase catalyzes the first step in glucose metabolism, using ATP for the phosphorylation of glucose to glucose-6-phosphate. Four different forms of hexokinase, designated type HK1, HK2, HK3, and HK4, are present in mammalian tissues.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8227
- synonyms:
